Source/NSParagraphStyle.m

```diff
@@ -303,6 +303,8 @@ - (id) init
           [_tabStops addObject: tab];
           RELEASE(tab);
         }
+      
+      ASSIGN(_textLists, [NSArray array]);
```

Comment:
Instances in Cocoa always have an empty array set for the default value, so I included this here and below.

We weren't even saving the textLists in some of the serialization paths, and other fields like text blocks and text tables are ignored as well. I'd suggest deferring any work on that part unless we make the changes as a batch and bump the object version; I'm not that familiar with any of this yet, and I don't want the PR to get out of hand.